src/commands/context/spec/context.spec.server.ts

```diff
@@ -93,11 +120,72 @@ describe('sasjs context', () => {
       },
       60 * 1000
     )
+
+    it(
+      'should return an error if trying to create compute context that already exists',
+      async () => {
```

Comment:
It's best to use a `done` callback here and avoid specifying an explicit timeout, since that is already set in the `jest.config.js` file.